pkg/apis/v1beta1/nodepool_validation_cel_test.go

```diff
@@ -105,15 +105,49 @@ var _ = Describe("CEL/Validation", func() {
 			nodePool.Spec.Disruption.Budgets = []Budget{{
```

Comment:
Should we be making the same test changes to the `nodepool_validation_webhook_test`?

Reply:
Will do in a follow up when I add those tests.

Reply:
I see. So we're not adding them right now because we don't yet have the webhook validations. Makes sense.